dataset/dictionary/objects.json5

```
@@ -279,6 +279,13 @@
     pronunciationJa: "せんぞう",
     tags: [ "object", "liyue" ],
   },
+  {
+    en: "Rattan Figure",
+    ja: "藤人形",
```

While 藤 with くさかんむりis used in Chinese version, 籐 with たけかんむり is used in Japanese version.

cf. [Lan Yan in HoYoWiki](https://wiki.hoyolab.com/m/genshin/entry/7301)

```suggestion
    ja: "籐人形",
```